packages/firebase_core/firebase_core/example/macos/.gitignore

```diff
@@ -0,0 +1,6 @@
+# Flutter-related
```

Comment:
You don't need a gitignore at this level. The root gitignore should be enough

Reply:
That's what `flutter create .` did. Maybe it should be removed from there then.

Reply:
Well `flutter create` doesn't know there's a root level .gitignore :)

Reply:
True :) But this one also ignores `**/Flutter/ephemeral/` and maybe more which is missing from the root level one.

Reply:
Should I add `**/Flutter/ephemeral/` to the root `.gitignore` or do we leave it as it is?

Reply:
Let's leave it as is right now. I'll add it to the root ignore and remove from here if needed in another PR